jobs/rocoto/atmanalprep.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash -x
+
+###############################################################
+# Source FV3GFS workflow modules
+. $HOMEgfs/ush/load_fv3gfs_modules.sh
+status=$?
+[[ $status -ne 0 ]] && exit $status
+
+###############################################################
+# set JJOB variables
+export configs="base atmanal atmanalprep"
+export EXSCRIPT=${GDASPREPPY:-$HOMEgfs/sorc/gdas.cd/scripts/exgdas_global_atmos_analysis_prep.py}
```

Comment:
The `EXSCRIPT` variable should be defined in `config.atmanlprep`.
I am still thinking more on the `configs` and what to do here.

Reply:
[6674720](https://github.com/NOAA-EMC/global-workflow/commit/667472055b8db74e05467f0c7b02eb6a67bce9d1) removed the `configs` and `EXSCRIPT` lines from `atmanalprep.sh`.  These lines are now in the function specific j-job, `JGDAS_GLOBAL_ATMOS_ANALYSIS_PREP`.